assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/multiuser.properties

```diff
@@ -185,3 +185,12 @@ che.keycloak.admin_username=NULL
 # from Che database. Make sense only in case ${che.keycloak.cascade_user_removal_enabled}
 # set to 'true'
 che.keycloak.admin_password=NULL
+
+# User name adjustment configuration. Represented by comma-separated key-value pairs. That is sequentially used
+# as arguments String.replaceAll function against original username.  Where 'key' is a regular expression.
+# Can be used to replace undesired characters like '\' when the user name is used as a part of k8s object for example.
+# Replaced user name would be stored in Che database only. It's recommended to use DNS-compliant values.
+# Value `NULL` is turning off this feature.
+# Example: '\\\\=-,@=-at-' This configuration will replace `\` to `-` and `@` to `-at-`. So `org\user@com` will looks
```

Comment:
Is the example actually correct? Wouldn't it have to be `\\=-,@=-at-` when written in the property file as opposed to the java string?

Reply:
you are right. Fixed.